requirements_dev.txt

```diff
@@ -5,3 +5,7 @@ sphinx>=1.7
 bump2version
 twine
 sphinx_rtd_theme
+nbval
+nbconvert
+birdhouse-birdy
+ipython          # needed for the parsing of Jupyter Notebooks
```

Comment:
Not sure why `ipython` is needed for parsing Jupyter Notebooks, have not seen it in Finch and Raven that have the same notebook testing mechanism.  But I don't mind having it here.

Reply:
@tlvu I took it from raven : https://github.com/Ouranosinc/raven/blob/master/requirements.txt#L37